docs/cuopt/source/cuopt-c/lp-milp/lp-milp-c-api.rst

```diff
@@ -153,15 +153,21 @@ These constants are used as parameter names in the :c:func:`cuOptSetParameter`,
 .. doxygendefine:: CUOPT_SAVE_BEST_PRIMAL_SO_FAR
 .. doxygendefine:: CUOPT_FIRST_PRIMAL_FEASIBLE
 .. doxygendefine:: CUOPT_LOG_FILE
+.. doxygendefine:: CUOPT_LOG_TO_CONSOLE
+.. doxygendefine:: CUOPT_CROSSOVER
+.. doxygendefine:: CUOPT_PRESOLVE
 .. doxygendefine:: CUOPT_MIP_ABSOLUTE_TOLERANCE
 .. doxygendefine:: CUOPT_MIP_RELATIVE_TOLERANCE
 .. doxygendefine:: CUOPT_MIP_INTEGRALITY_TOLERANCE
-.. doxygendefine:: CUOPT_MIP_SCALING
+.. doxygendefine:: CUOPT_MIP_ABSOLUTE_GAP
```

Comment:
Do we need to update this name change in settings doc and anywhere else as well ?

Reply:
This is likely a merge conflict issue on my side. Only presolve should be added here.